app/src/main/res/values/strings.xml

```diff
@@ -94,4 +94,5 @@
     <string name="invalid_token_error">Invalid or expired credentials</string>
     <string name="category_up_description">Navigate to parent category</string>
     <string name="loading_child_categories">Fetching categories</string>
+    <string name="no_listings_available">There are no listings in this category</string>
```

Comment:
Since the search results are also determined by other filters, we should probably make the string more generic. How about displaying something along the lines of:

**No items found**
We couldn't find anything matching your query. You could try adjusting your filters or search inside a different category.

Reply:
- [x]

Reply:
- [ ]